ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/internalresponse/EthAccountsResultProvider.java

```diff
@@ -45,7 +44,17 @@ public EthAccountsResultProvider(final Supplier<Set<String>> addressesSupplier)
       throw new JsonRpcException(JsonRpcError.INVALID_PARAMS);
     }
 
-    return addressesSupplier.get().stream().sorted().collect(Collectors.toList());
+    final List<String> availableAddress = Lists.newArrayList();
```

Comment:
This can probably be simplified to just do the filter on the stream
i.e.
```return transactionSignerProvider.availableAddresses().stream()
        .sorted()
        .filter(addr -> transactionSignerProvider.getSigner(addr).isPresent())
        .collect(Collectors.toList());```